source/core/parameters_lagrangian.cc

```diff
@@ -1199,15 +1193,24 @@ namespace Parameters
         }
       else if (boundary_type == "rotational")
         {
-          BC_type                       = BoundaryType::rotational;
-          double       rotational_speed = prm.get_double("rotational speed");
-          Tensor<1, 3> rotational_vector;
-
-          rotational_vector[0] = prm.get_double("rotational vector x");
-          rotational_vector[1] = prm.get_double("rotational vector y");
-          rotational_vector[2] = prm.get_double("rotational vector z");
+          BC_type                 = BoundaryType::rotational;
+          double rotational_speed = prm.get_double("rotational speed");
 
+          // Read the rotational vector from a list of doubles
```

Comment:
Not necessary to do it now, but we should clearly do something like a string_to_tensor<dim> function to refactor this parsing.

Reply:
I'll do the change in an other PR. Good idea